packages/components/src/components/Tabs.vue

```diff
@@ -131,36 +135,35 @@ export default {
   }
 }
 
-@media (max-width: 1100px) {
-  .tabs {
-    ::-webkit-scrollbar {
-      display: none;
-    }
-    &__header {
-      flex-direction: column-reverse;
-      min-height: 70px;
-      padding-top: 0.5rem;
-      height: auto;
-      gap: 0.4rem;
-      justify-content: end;
-      overflow-x: scroll;
-      overflow-y: hidden;
-    }
-    &__group {
-      align-self: center;
-      justify-content: center;
-    }
-    &__notification {
-      align-self: center;
-    }
-    &__content {
-      padding-top: 70px;
-    }
-    &__controls {
-      gap: 1rem;
-      position: fixed;
-      top: 0.5rem;
-    }
+.narrow {
```

Comment:
Can this be a container query instead? Using `.tabs` as the container, this can all flow reactively in pure CSS.

e.g. (shortened for brevity)
```scss
.tabs {
  container-type: inline-size;
}

@container (max-width: 600px) {
  // reactive styles here
}
```

Reply:
Is this too bleeding edge?
https://caniuse.com/css-container-queries

Reply:
I originally tried to fix this when I was working on the search bar and I used the `@container` query, but I ran into problems. I don't remember what they were, but I'll give it another try.

Reply:
It's causing a weird rendering problem, so I think we shouldn't use the container queries:

![image](https://github.com/getappmap/appmap-js/assets/45714532/c515344b-c34b-4918-93a6-1decbde4283c)